gradle.properties

```diff
@@ -1,10 +1,17 @@
-name = IdeaJasmine
-group = io.pivotal.jasmine
-version = 1.10
-ideaVersion = IU-2022.2.5
+pluginName = IdeaJasmine
+pluginGroup = io.pivotal.jasmine
+pluginVersion = 1.11
 publishChannels = Stable
+pluginRepositoryUrl = https://github.com/jasmine/IdeaJasmine
+pluginDescription = Run your Jasmine 3.0+ tests right from Intelli-J
 
-kotlin.stdlib.default.dependency = false
+# Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
+pluginSinceBuild = 242
```

Comment:
Are you sure you were able to install the plugin in WebStorm 2023.x? This _should_ mean that 2023.x is too old, and indeed I get "Plugin 'IdeaJasmine' (version '1.11') is not compatible with the current version of the IDE, because it requires build 242 or newer but the current build is WS-233.15619.15" when I try to install in 2023.3.8.

I don't think dropping support for 2023.x is a problem. Just wondering if there's something I'm missing here.

Reply:
> I don't think dropping support for 2023.x is a problem. Just wondering if there's something I'm missing here.

I don't know anything about the publish process, but the IDEA plugin system changed a lot in 2025.1, we won't be able to make it backwards compatible. Is it possible to publish multiple versions for different IDEA ranges?

Also, I took out the pluginUntilBuild range because I don't want to arbitrarily prevent the plugin from working in future versions when we don't know at which point IDEA will be backwards incompatible for IdeaJasmine.